kraken/network_chaos/actions.py

```diff
@@ -0,0 +1,213 @@
+import yaml
+import logging
+import time
+import kraken.cerberus.setup as cerberus
+import kraken.kubernetes.client as kubecli
+import kraken.node_actions.common_node_functions as common_node_functions
+import sys
+
+
+job_yml = """
```

Comment:
Can we add these as actual files somewhere in the repo, maybe in the network_scenario folder. Would just look cleaner

Reply:
sure will do, thanks for the suggestion, will add them as jinja templates in the network_chaos folder.